.github/workflows/ci.yml

```diff
@@ -57,7 +57,6 @@ jobs:
         run: |
           ./gradlew --version
           echo ANDROID_SDK_ROOT: ${ANDROID_SDK_ROOT}
-          ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --version
```

Comment:
I mean, it's not really doing much, just printing the version of the sdkmanager (aparently) this task is printing environment info.
And right now this was breaking CI, so the fastest solution was to remove this print.

why CI was broken? 
the sdkmanager isn't anymore in that folder (`tools/bin`)
@alif found it in `cmdline-tools/latest/bin`, but this version of the sdkmanager is also breaking CI
because it has been build with a newer version of Java
![Captura de pantalla 2024-05-16 a la(s) 12 25 02 a  m](https://github.com/rollbar/rollbar-java/assets/31810096/c9528da5-3ffc-4551-8a82-5b45918398fa)